Analysis/R/prepare_stan_input.R

```diff
@@ -332,19 +334,19 @@ prepare_stan_input <- function(
   }
   
   # Overwrite tfrac with user-specified value
-  if (!is.null(set_tfrac)) {
+  if (!is.null(set_tfrac) && (set_tfrac)) {
     cat("-- Overwriting tfrac with user-specified value of ", set_tfrac)
-    ind_mapping_resized$tfrac <- rep(set_tfrac, length(ind_mapping_resized$tfrac))
+    ind_mapping_resized$tfrac <- rep(1.0, length(ind_mapping_resized$tfrac))
```

Comment:
this seems funny - is this right? if set_tfrac is yes wouldn't you want to use tfrac_thresh?

Reply:
if set_tfrac is true, then always set tfrac to 1.0. That's the logic here right?